### PR TITLE
Update endpoint-dlp-using.md

### PR DESCRIPTION
**Pull request by FBrownMSFT**
add important note to Unallowed browsers

### CODE REVIEW

**Comment by yogkumgit** [comment]
dlp-sensitivity-label-as-condition.md

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/endpoint-dlp-using.md [hunk 1]
```diff
@@ -65,7 +65,10 @@ When a policy's **Access by unallowed apps and browsers** setting is turned on a
 
 ### Unallowed browsers
 
-You add browsers, identified by their process names, that will be blocked from accessing files that match the conditions of an enforced  a DLP policy where the upload to cloud services restriction is set to block or block override. When these browsers are blocked from accessing a file, the end users will see a toast notification asking them to open the file through Edge Chromium.
+You add browsers, identified by their executable names, that will be blocked from accessing files that match the conditions of an enforced  a DLP policy where the upload to cloud services restriction is set to block or block override. When these browsers are blocked from accessing a file, the end users will see a toast notification asking them to open the file through Edge Chromium.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/endpoint-dlp-using.md]:
```suggestion
You add browsers, identified by their executable names, that will be blocked from accessing files that match the conditions of an enforced a DLP policy where the upload to cloud services restriction is set to block or block override. When these browsers are blocked from accessing a file, the end users will see a toast notification asking them to open the file through Edge Chromium.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - one minor additional edit (please remove extra space).

**Comment by yogkumgit** [comment]
@FBrownMSFT Please implement the review comment. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks.

**Pull request merged**